mathics/builtin/drawing/image_internals.py

```diff
@@ -78,7 +78,7 @@ def pixels_as_ubyte(pixels):
         return pixels
     elif dtype == numpy.uint16:
         return (pixels / 256).astype(numpy.uint8)
-    elif dtype == numpy.bool:
+    elif dtype == bool:
         return pixels.astype(numpy.uint8) * 255
```

Comment:
Why equal instead of is?

Reply:
Also, I guess is a good ocasion to move this to the mathics.eval module.

Reply:
> Why equal instead of is?

Fixed now.

> Also, I guess is a good occasion to move this to the mathics.eval module.

Yes, easy enough done.

Reply:
And done now.